packages/components/app/styles/components/alert.scss

```diff
@@ -58,3 +57,111 @@
     }
   }
 }
+
+//
+// COLOR & TYPE
+//
+.hds-alert--color-critical {
+  background-color: var(--token-color-surface-critical);
+
+  &.hds-alert--type-page {
+    box-shadow: 0px 1px 0px 0px var(--token-color-border-critical);
+  }
+
+  &.hds-alert--type-inline { 
+    border-color: var(--token-color-border-critical);
+  }
+
+  .hds-alert__icon, .hds-alert__title {
+    color: var(--token-color-foreground-critical-on-surface);
+  } 
+}
+
+.hds-alert--color-warning {
+  background-color: var(--token-color-surface-warning);
+
+  &.hds-alert--type-page {
+    box-shadow: 0px 1px 0px 0px var(--token-color-border-warning);
+  }
+
+  &.hds-alert--type-inline { 
+    border-color: var(--token-color-border-warning);
+  }
+
+  .hds-alert__icon, .hds-alert__title {
+    color: var(--token-color-foreground-warning-on-surface);
+  }
+}
+
+// TODO: Check all neutral tokens, cause don't follow pattern of other `color`s 
```

Comment:
re: This TODO

Noticed the design tokens for `neutral` didn't follow the same naming pattern as the others e.g. `var(--token-color-foreground-warning-on-surface)` for `warning`, is that right? @didoo @cveigt

Reply:
thanks, sounds good 0350cb1dce06bc08923453f180715d4b35f34775